app/js/components/mixins/ModalEffects.react.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+let ModalEffects = InnerComponent => class extends React.Component {
```

Comment:
We should be consistent int he way we structure our react component so that it will be easily readable for those reading the code
This could have been implemnted like this

```javascript
class InnerComponent extends React.Component {
// your codes here
}
export default InnerComponet 
```
this conforms with the other implementation of the components